packages/config/src/common/ProjectRisk.ts

```diff
@@ -16,7 +16,7 @@ export type ProjectRiskCategory =
   | 'Users can be censored if'
   | 'MEV can be extracted if'
 
-export type Sentiment = 'bad' | 'warning' | 'good' | 'neutral'
+export type Sentiment = 'bad' | 'warning' | 'good' | 'neutral' | 'UnderReview'
```

Comment:
I think that I came up with this name `UnderReview` but the more I see it I think it should be `under-review`. I will create a task to rename it in the future.

Reply:
100%